lib/Admin.php

```diff
@@ -6,13 +6,25 @@ class Admin {
 
 	public static function init() {
 		$filter = add_filter('plugin_row_meta', array(__CLASS__, 'meta_links'), 10, 2);
+		$action = add_action('after_plugin_row_timber-library/timber.php', array(__CLASS__, 'show_deprecation_warning'), 10, 3);
 		$action = add_action('in_plugin_update_message-timber-library/timber.php', array(__CLASS__, 'in_plugin_update_message'), 10, 2);
 		$action = add_action('in_plugin_update_message-timber/timber.php', array(__CLASS__, 'in_plugin_update_message'), 10, 2);
 		if ( $filter && $action ) {
 			return true;
 		}
 	}
 
+	/**
+	 * show additional information about the plugin deprecation.
+	 *
+	 * @param string  $file
+	 * @param array   $plugin
+	 */
+	public static function show_deprecation_warning($file, $plugin, $status)
+	{
+		echo '<td colspan="3" class="plugin-update colspanchange"><div class="update-message notice-error notice inline notice-warning notice-alt"><p>We recommend <a target="_blank" href="#">switching to the Composer based version of Timber</a>. <a href="#" target="_blank">Why you need to switch.</a></p></div></td>';
```

Comment:
I’d make the first link take fewer words so that it’s easier to see that it’s two links.

```suggestion
		echo '<td colspan="3" class="plugin-update colspanchange"><div class="update-message notice-error notice inline notice-warning notice-alt"><p>We recommend <a target="_blank" href="#">switching to the Composer based version</a> of Timber. <a href="#" target="_blank">Why you need to switch.</a></p></div></td>';
```

Reply:
Nice, added the link as well @gchtr